Cards/CardContainer.gd

```diff
@@ -61,11 +61,16 @@ func set_queued_card(card: CardWorld) -> void:
 
 
 func remove_queued_card() -> void:
```

Comment:
this name might actually be a bit confusing
`remove_queued_card` I would expect that to be when you decide to use another card, Like I select card A, it gets queued, but then I select card B to play it instead, and I remove A from the queue
Maybe `set_awaiting_play` would be more appropriate ? Because from what I understand the card does
`hand -> queued -> queued for discard -> active -> discard`
I also think `queued for discard -> active` is a bit confusing in itself, why this choice ?

Reply:
I'll wait for #39 to get merged in, since there are some changes Jona made to these lines indirectly. I will handle the merge conflicts and address this then!

But you're right, the naming should be better for this phase of the card flow. I'll bring it back to the drawing board.

Reply:
naming things is always hard anyway

Reply:
You got a good point about that `queued for discard -> active` I was thinking of making it `ready` instead?

The flow would then be:
`hand -> queued -> ready -> active -> discard`

And in the ready state we remove it from the `cards_in_hand` list

Reply:
In terms of the naming, I was thinking of moving from `remove_queued_card` to `set_queued_card_to_ready_card`. Thoughts?

Reply:
or maybe `queued_for_active` ? what was the plan with having a `queued_for_discard` in the first place ? a timer to make sure everything is properly discarded ?